libsolidity/inlineasm/AsmParser.cpp

```diff
@@ -209,7 +214,11 @@ assembly::Statement Parser::parseElementaryOperation(bool _onlySinglePusher)
 		break;
 	}
 	default:
-		fatalParserError("Expected elementary inline assembly operation.");
+		fatalParserError(
+			m_julia ?
+			"Expected literal or identifier." :
+			"Expected elementary inline assembly operation."
```

Comment:
So which order do we want, expected in front or back? It is kind of random at the moment.